packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/index.md

```diff
@@ -27,7 +27,7 @@ In this guide you will learn how to integrate biometric authentication like Face
 
 ## Overview
 
-When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/configure-packages/) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
+When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/android/main/#add-and-configure-packages) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
```

Comment:
```suggestion
When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/-/main/#add-and-configure-packages) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
```
Both this page and the /sign-into-mobile-app/ page contain the same stack selectors, so we can't hardcode "android".